src/parsec.rs

```diff
@@ -649,13 +649,13 @@ impl<T: NetworkEvent, S: SecretId> Parsec<T, S> {
         match *payload {
             Observation::Add(ref other_peer_id) => self
                 .peer_list
-                .add_peers_peer(peer_id, other_peer_id.clone()),
+                .add_to_membership_list(peer_id, other_peer_id.clone()),
```

Comment:
```suggestion
                .add_to_peers_membership_list(peer_id, other_peer_id.clone()),
```
It would make it more obvious what both arguments mean. As is, you could think you're adding two peers to some membership list.